### ISSUE TITLE
Implement the Ajax status messages in upload.php

### ISSUE THREAD

**Issue by AJM-1** [computationalmystic/RJI-group23#16]
Each file to be uploaded should be uploaded individually
after the files are all done succeeding or failing, a request to analyze.php should run the script for all the images uploaded and display a success message

**Comment by AJM-1** [comment]
This is done, but not with ajax. Instead, the files are uploaded all at once, but status messages are displayed for each file.
There doesn't seem to be an advantage uploaded each one individually beside one bug where some files cause an array error in the upload

**Comment by AJM-1** [comment]
upload.php doesn't upload each image individually with ajax but it does fetch the assessment with ajax. That is all we will be implementing at this point
